Bugsnag/KSCrash/Source/KSCrash/Recording/BSG_KSCrash.m

```diff
@@ -271,21 +267,6 @@ - (BOOL)install {
     return true;
 }
 
-/**
- * Set up listeners for un/loaded frameworks.  Maintaining our own list of framework Mach
- * headers means that we avoid potential deadlock situations where we try and suspend
- * lock-holding threads prior to loading mach headers as part of our normal event handling
- * behaviour.
- */
```

Comment:
nit: would be good to retain this comment at the first invocation of `bsg_mach_headers_initialize`

Reply:
It was a bit detailed for a comment outside the mach-headers source, but I've added it to the header and made a more general comment here.